libs/frontend/feature-build/src/lib/use-padding-margin-edit.ts

```diff
@@ -120,6 +120,9 @@ const drag = (e: MouseEvent) => {
           dragging.value.startY = e.clientY
         }
         const replace = !firstMove
+        // TODO -- to avoid the issue where undo sets the style back to an inherited
+        // value instead of removing it, we need to maintain `oldStyle: undefined` when
+        // replacing the command
```

Comment:
Unfortunately I didn't have time to get this to work for the margin/padding editor UI. The code is older so it's not using the new style helpers, but should be possible to fix.